src/libraries/Pausable.sol

```diff
@@ -49,6 +84,21 @@ abstract contract Pausable is Initializable {
         _getPauseStorage()._pauseFlag = pauseFlag;
     }
 
+    function __Paused_init(address initialPauser, address owner) internal onlyInitializing {
+        __Paused_init_unchained(initialPauser, owner);
+    }
+
+    function __Paused_init_unchained(
```

Comment:
On init I don't think we need to worry about the owner do we? Can we just set the initial pauser directly rather than calling `_transferPauserCapability`?